pkg/components/logging/component.go

```diff
@@ -14,7 +14,26 @@ var LoggingComponent = Component{
 		Name:                 "Logging",
 		Operators:            []string{},
 		DefaultJiraComponent: "Logging",
-		Matchers:             []config.ComponentMatcher{},
+		Matchers: []config.ComponentMatcher{
+			{Suite: "Elasticsearch related tests"},
+			{Suite: "Kibana related features"},
+			{Suite: "LOGGING"},
+			{Suite: "Logging"},
+			{Suite: "Logging related features"},
+			{Suite: "Logging smoke test case"},
+			{Suite: "Logging view on the openshift console"},
+			{Suite: "cluster log forwarder testing"},
+			{Suite: "cluster logging related scenarios"},
+			{Suite: "cluster-logging-operator related cases"},
+			{Suite: "cluster-logging-operator related test"},
+			{Suite: "elasticsearch operator related tests"},
+			{Suite: "elasticsearch related tests"},
+			{Suite: "elasticsearch-operator related tests"},
+			{Suite: "logs related features"},
+			{Suite: "Cases to test forward logs to external elasticsearch"},
+			{Suite: "cluster log forwarder features"},
+			{Suite: "fluentd related tests"},
+		},
```

Comment:
I don't see feature below in Suite? shall use use these items>

verification-tests repo:
collector related tests
Elasticsearch related tests
eventrouter related test
Cases to test forward logs to external elasticsearch
permission related test

cucushift repo:
log forwarding related tests
logging permission related tests
kibana web UI related cases for logging